server/routes/api/user.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const userModel = require('../../models/User');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const config = require('config');
+const router = express.Router();
+const {check, validationResult} = require('express-validator');
+
+//api for user registeration/creation
+router.post('/register',
+[
```

Comment:
what IDE are you using to write code? perhaps look into a linter. this will help code be more organized and easier to read :)